project-base/tests/App/Acceptance/acceptance/PageObject/Front/RegistrationPage.php

```diff
@@ -6,10 +6,31 @@
 
 use Shopsys\FrameworkBundle\Component\Form\TimedFormTypeExtension;
 use Tests\App\Acceptance\acceptance\PageObject\AbstractPage;
+use Tests\App\Test\Codeception\AcceptanceTester;
+use Tests\App\Test\Codeception\Module\StrictWebDriver;
 use Tests\FrameworkBundle\Test\Codeception\FrontCheckbox;
 
 class RegistrationPage extends AbstractPage
 {
+    protected const MINIMUM_FORM_SUBMIT_WAIT_TIME = 10;
```

Comment:
This constant is copy-pasted in several other classes. Does it make sense to unify it?

Reply:
I have removed excessive const from OrderCest